contrib/win32/openssh/OpenSSHBuildHelper.psm1

```diff
@@ -344,6 +345,10 @@ function Package-OpenSSH
         }
     }
 
+    #copy libcrypto-41 dll
+    $libreSSLSDKPath = Join-Path $PSScriptRoot $script:libreSSLSDKStr
+    Copy-Item -Path $(Join-Path $libreSSLSDKPath "$NativeHostArch\libcrypto-41.dll") -Destination $packageDir -Force -ErrorAction Stop    
```

Comment:
DestinationPath is getting cleared right below. You'll need to move this post that.